docs/aws/README.md

```diff
@@ -21,6 +21,7 @@ Karpenter can run anywhere, including on self-managed node groups, [managed node
 This demo will run Karpenter on Fargate, which means all EC2 instances added to this cluster will be controlled by Karpenter.
 
 ```bash
+# If the following URL gives a 404, use https://raw.githubusercontent.com/awslabs/karpenter/main/docs/aws/eks-config.yaml
 curl -fsSL https://raw.githubusercontent.com/awslabs/karpenter/"${KARPENTER_VERSION}"/docs/aws/eks-config.yaml \
```

Comment:
I think you should be able to do something like ${KARPENTER_VERSION:-main} which will default to main if not set.

Reply:
Yes, but we tell the user to set this above, so that means this will be set if they followed the guide from top to bottom.